### ISSUE TITLE
Ability to retire/archive a counter

### ISSUE THREAD

**Issue by dariaGirl** [albertvaka/bettercounter#31]
Great app.
Popping in to request the ability to retire/archive a counter. 

Which i guess would be some sort of auto-export/Import, store CSV in app folder, connected to a specifically named counter stored in the app.

**Comment by albertvaka** [comment]
So you would like to export a single counter before deleting it, right? It makes sense, but I'm not sure where this could fit in the UI.

Some brainstorming: Maybe we could add it as a question after you click "delete" (eg: "do you want to export this counter before deleting it?"). Or maybe we could separately add the option to export a single counter from the three dots menu... then you could use that option before deleting one, or in any other moment (which is a bit more versatile, but requires two actions for your use case: first export, then delete).

**Comment by MikeStirner** [comment]
Or to make it a more broad feature, the app could allow the user to move a counter off of the main list and onto another list. There could be an "Archived" list by default but the user can add more. Would also help people like me who have 100 counters from needing to scroll so far all the time haha.

And when you export data, the CSV could look like: counter_name,list_name,epochtime1,epochtime2,...

**Comment by albertvaka** [comment]
Yes, other people already requested categories/tabs. It could be used as a workaround for archived/old counters.

Although "retired" counter management is more complex than just that... Eg: the last time I retired a counter I actually wanted to merge it with another one. I don't think this is common enough that we need to support it in-app, though (given you can always export, modify & re-import) , but I'm not sure.

**Comment by MikeStirner** [comment]
Yeah, I've only wanted to do that one time so far, where I wanted to basically reset a counter, but I wanted to keep the original timestamps of the times I used that counter. But I just worked around it by doing an export, and then resetting the counter. And that's good enough for me.

And if I wanted to merge, then I'd probably just do an export. And I could probably figure it out in 10 to 20 minutes, and then re-import. So, probably not worth doing a bunch of coding when it might only happen for someone every month or two or never.

If I ever do this, in spreadsheet format, some kind of merge function, maybe I'll post it as a file here, and then people can just use that spreadsheet.

Another workaround is to have two different counters, so that you don't have to reset one of the counters. One of the counters is the number going up, and one of the counters is the number going down, and so then in your head you just subtract one from the other, and that difference is the current number.

An example would be... How many yogurts do I have in the fridge? And so I have a counter that counts how many yogurts I've purchased ever. And another counter next to it representing how many yogurts I've consumed from the fridge ever.

**Comment by nvllz** [comment]
@albertvaka what about just hiding the entry from the list when archived? I have a few old counters I would like to keep for the future, but they're already complete and irrevelant. Just a checkbox "Show Archived” in the three-dot-menu to toggle the visibility state and a checkbox/switch in the Edit counter dialog to manage the counter state.

**Comment by albertvaka** [comment]
@nvllz As a workaround, you can export the counter and if you ever want to see it again re-import it. I know it's not as convenient, but should work.

**Comment by nvllz** [comment]
@albertvaka I mean.. Wanted to ask if you even plan to add it this way? I set it to export automatically on change so this would not work for me.

Like I have two counters: goal2025–300x and goal2026–400x. As I decided I can do better this year year, I had to make a new counter with a higher value. But as this applies to at least two of my last year's goals, my counter list is a bit cluttered.

If it's not planned, I'm sticking to my way for now. But thanks for your answer.

**Comment by albertvaka** [comment]
Oh, I thought you could export a single counter but apparently that's not an option 😅 

I don't have a plan to implement archiving counters as of now, mostly because it means adding more buttons to the UI and making it more complex for a feature that is rarely used.
